src/main/java/seedu/address/logic/commands/MarkAllStudentAbsentCommand.java

```diff
@@ -47,7 +47,7 @@ public CommandResult execute(Model model) throws CommandException {
             try {
                 model.markStudentAbsent(studentToUnmark, studentClass, editedStudent);
                 model.updateFilteredClassList((c) -> c.isSameClass(studentClass));
-            } catch (StudentAlreadyMarkedPresent e) {
+            } catch (StudentAlreadyMarkedAbsent e) {
```

Comment:
Thanks for creating custom exceptions! But just a nitpick, maybe it would be better to end the name with `Exception` to differentiate it from a normal Java class, although I don't think there is a standard for Exception file naming?

Reply:
Thanks for pointing it out! Naming with Exception definitely is better 👍 Will make the changes